### PR TITLE
Update German translation for PR #1558

### PR DESCRIPTION
**Pull request by FriedrichFroebel**
Adds translations for the strings from #1558.

### CODE REVIEW

File: src/translations/sqlb_de.ts [hunk 1]
```diff
@@ -5337,12 +5337,12 @@ Kann auf 0 gesetzt werden, um die Vervollständigung zu deaktivieren.</translati
...
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;SQLite biete eine SQL-Funktion an, um Erweiterungen aus einer Shared-Library-Datei zu laden. Aktivieren Sie dies, falls Sie die &lt;span style=&quot; font-style:italic;&quot;&gt;load_extension()&lt;/span&gt;-Funktion aus SQL-Code heraus benutzen möchten.&lt;/p&gt;&lt;p&gt;Aus Sicherheitsgründen ist das Laden von Erweiterungen standardmäßig deaktiviert und muss durch diese Einstellung aktiviert werden. Sie können alternativ immer die gewünschten Erweiterungen über die GUI laden, auch wenn diese Option deaktiviert ist.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
```
[the remark this answers is not in the document]
Comment (FriedrichFroebel) [reply src/translations/sqlb_de.ts]: In fact this is a typo.

**Comment by FriedrichFroebel** [comment]
Travis seems to have some problems with fetching the needed packages.

**Comment by justinclift** [comment]
Yeah, the Travis infrastructure seems to go through periods of being flaky.  Doesn't seem to be a cause for concern in itself, as it generally improves a day or two later. :smile:

**Comment by mgrojo** [comment]
Thanks @FriedrichFroebel. I'll merge this to master and then I'll cherry-pick it to the release branch.

**Pull request merged**